spec/lib/mavenlink/workspace_spec.rb

```diff
@@ -410,21 +410,17 @@
 
         specify do
           expect do
-            begin
-                     subject.save!
-            rescue StandardError
-              nil
-                   end
+            subject.save!
+          rescue StandardError
+            nil
```

Comment:
I would think the `begin` block is necessary here. Otherwise doesn't it always return `nil`?

Reply:
Rubocop actually made me change it to that. https://www.rubydoc.info/gems/rubocop/0.27.0/RuboCop/Cop/Style/RedundantBegin

Reply:
yea, there might be a special syntax case for rescue without begin...

you can also do stuff like

```
def thang(args)
  # ... stuff that might raise exception
rescue ExceptionBit
  # ... yadda
end
```